releasenotes/notes/imag_gradients-3dabcd11343062a8.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    Enable the calculation of the imaginary part of expectation value gradients. 
+    Using a different measurement basis, i.e. ``-Y`` instead of ``Z``, we can measure the
+    imaginary part of gradients using the existing :class:`~qiskit.circuit.opflow.Gradient` and
+    :class:`~qiskit.circuit.opflow.QFI` classes. 
+    The measurement basis can be set with the ``aux_meas_op`` argument.
```

Comment:
Could you add a code snippet that shows how to use the new feature? That's always super helpful for the release notes to show people an example of what we added 🙂